olad/RDMHTTPModule.cpp

```diff
@@ -3418,7 +3418,7 @@ void RDMHTTPModule::GetCurveDescriptionHandler(
     description = resp_description;
   }
 
-  info->curves.push_back(pair<uint32_t, string>(curve, description));
+  info->curve_descriptions.push_back(description);
```

Comment:
So now you've made this change, the curve parameter should now be OLA_UNUSED again!